bin/fin

```diff
@@ -4720,13 +4720,13 @@ ngrok_share ()
 	port=${port:-${NGROK_PORT:-80}}
 
 	# Set the protocol.
-	if [[ "$protocol" = "TCP" ]]; then
+	if [[ "$protocol" == "TCP" ]]; then
 	  ARGS="$ARGS tcp"
 	else
 	  ARGS="$ARGS http"
 	fi
 
-	NGROK_PORT=$(echo ${port} | sed 's|^tcp://||')
+	NGROK_PORT=$(port#tcp://)
```

Comment:
It's curly braces https://github.com/docksal/docksal/pull/604#discussion_r194706659
Parenthesis is a subshell